metal-vhost builds fail

**What happened:**

When creating a PR, the `metal-vhost` flavor currently fails.

**Debug Logs:**

https://github.com/gardenlinux/gardenlinux/actions/runs/11103774677/job/30846622027?pr=2374

```
remote_files = ['/usr/bin/chfn,root,root', '/usr/bin/chsh,root,root', '/usr/bin/gpasswd,root,root', '/usr/bin/newgrp,root,root', '/usr/bin/passwd,root,root', '/usr/bin/pkexec,root,root', ...]
whitelist_files = ['/usr/bin/chsh,root,root', '/usr/lib/openssh/ssh-keysign,root,root', '/usr/bin/newgrp,root,root', '/usr/bin/su,root,root', '/usr/lib/dbus-1.0/dbus-daemon-launch-helper,root,messagebus', '/usr/bin/chfn,root,root', ...]

    def _val_whitelist_files(remote_files, whitelist_files):
        """ Validates that remotly found files are in whitelist """
        found_files = []
        for file in remote_files:
            if file not in whitelist_files:
                found_files.append(file)
>       assert not found_files, f"{found_files}"
E       AssertionError: ['/usr/sbin/mount.nfs,root,root']
```

**How to reproduce it**:

Run a build with the `metal-vhost` flavor/feature enabled.

--- a/features/base/test/test_sgid_suid_files.py
+++ b/features/base/test/test_sgid_suid_files.py
@@ -27,7 +27,8 @@ import pytest
                  "/usr/bin/sudo,root,root",
                  "/usr/bin/passwd,root,root",
                  "/usr/lib/polkit-1/polkit-agent-helper-1,root,root",
-                 "/usr/bin/pkexec,root,root"
+                 "/usr/bin/pkexec,root,root",
+                 "/usr/sbin/mount.nfs,root,root"
                  ]
         )
     ]

